develop/components/pages/TooltipPage.tsx

```diff
@@ -1,22 +1,81 @@
 
 const TooltipPage: React.FunctionComponent = () => {
+    const contentList: Array<RadioListModel<number>> = [
+        { label: "content", value: 0 },
+        { label: "message", value: 1 },
+        { label: "messageGroup", value: 2 }
+    ];
+    const triggerList: Array<RadioListModel<TooltipTrigger>> = [
+        { label: "Click", value: "click" },
+        { label: "Hover", value: "hover" },
+        { label: "Focus", value: "focus" }
+    ];
+    const themeList: Array<RadioListModel<TooltipTheme>> = [
+        { label: "Primary", value: "primary" },
+        { label: "Danger", value: "danger" },
+        { label: "Default", value: "default" },
+        { label: "Light", value: "light" },
+        { label: "Purple", value: "purple" },
+        { label: "Success", value: "success" },
+        { label: "Warning", value: "warning" },
+    ];
+    const positionList: Array<RadioListModel<TooltipPosition>> = [
+        { label: "Top", value: "top" },
+        { label: "Top Left", value: "top-left" },
+        { label: "Top Right", value: "top-right" },
+        { label: "Right", value: "right" },
+        { label: "Right Top", value: "right-top" },
+        { label: "Right Bottom", value: "right-bottom" },
+        { label: "Bottom", value: "bottom" },
+        { label: "Bottom Left", value: "bottom-left" },
+        { label: "Bottom Right", value: "bottom-right" },
+        { label: "Left", value: "left" },
+        { label: "Left Top", value: "left-top" },
+        { label: "Left Bottom", value: "left-bottom" },
+    ];
+    const defaultMessageGroup: Array<TooltipMessageGroupItem> = [
+        {
+            title: "Tooltip title",
+            message: "tooltip 1"
+        },
+        {
+            message: "tooltip 2"
+        }
+    ];
+    const defaultTooltipContent: string = "Tooltip content could be long, therefore, controlling the position and width is important";
+    const nodeTooltipContent: React.ReactNode = (<div><h1>Tooltip Header</h1><div>tooltip content</div></div>);
+    const [theme, setTheme] = React.useState<TooltipTheme>("primary");
```

Comment:
would it be possible to use the value that you created in themeList, same thing for position and trigger

Reply:
updated the value using the list